### ISSUE TITLE
Loosen requirements versions for installation as a package; python-Levenshtein==0.12.0 is insecure

### ISSUE THREAD

**Issue by AndydeCleyre** [soxoj/socid-extractor#53]
```console
$ printf '%s\n' socid-extractor >reqs.txt
$ pip install -Uqr reqs.txt
WARNING: The candidate selected for download or install is a yanked version: 'python-levenshtein' candidate (version 0.12.0 at https://files.pythonhosted.org/packages/42/a9/d1785c85ebf9b7dfacd08938dd028209c34a0ea3b1bcdb895208bd40a67d/python-Levenshtein-0.12.0.tar.gz#sha256=033a11de5e3d19ea25c9302d11224e1a1898fe5abd23c61c7c360c25195e3eb1 (from https://pypi.org/simple/python-levenshtein/))
Reason for being yanked: Insecure, upgrade to 0.12.1
```

I don't think the PyPI package should be so strict about exact dependency versions, so that your package no longer enforces an insecure setup upon installation.

**Comment by soxoj** [comment]
Sure, you are completely right. I will fix it.

**Comment by soxoj** [comment]
I've published v0.0.12 with non-strict dependency versions, please check it.
https://pypi.org/project/socid-extractor/0.0.12/

**Comment by AndydeCleyre** [comment]
Works great, thanks!
